prow/gerrit/adapter/adapter_test.go

```diff
@@ -513,6 +513,24 @@ func TestProcessChange(t *testing.T) {
 			instance:     testInstance,
 			wantError:    true,
 		},
+		{
```

Comment:
will need more tests based on different comments, including corner cases of comments like
```
- /test something
```

Reply:
Done.